app/Http/Controllers/EventController.php

```diff
@@ -90,7 +91,37 @@ public function show($id, Request $request)
      */
     public function update(Request $request, $id)
     {
-        return  response()->json(['status' => 'error', 'message' => 'method_not_implemented'], 501);
+        $event = Event::find($id);
+
+        if (isset($request->organizer)) {
+            $organizer = UserController::getUserByIdentifier($request->organizer);
```

Comment:
After merge of #158, please change this to reflect the newly-created Eloquent scope for searching by identifier.